patches/chrome-browser-resources-settings-controls-controlled_radio_button.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/settings/controls/controlled_radio_button.html b/chrome/browser/resources/settings/controls/controlled_radio_button.html
+index b2c2c288a5fe0940f99843a4f556cfb4d73ef5bd..e9095c6c81921cd861cd5a6b1b702ee8ba5845ee 100644
+--- a/chrome/browser/resources/settings/controls/controlled_radio_button.html
++++ b/chrome/browser/resources/settings/controls/controlled_radio_button.html
+@@ -1,6 +1,6 @@
+ <style
+ <if expr="not chromeos_ash">
+-  include="cr-radio-button-style"
++  include="settings-shared cr-radio-button-style"
```

Comment:
@petemill should we override cr-radio-button-style instead of patching in settings-shared?

Reply:
@petemill is on PTO the rest of this week, but I did a fixup to include `br-shared-style` in `cr_radio_button_style.css` here 71eea47a764524b42b7976f9df9f2e98b07d7859. I'll touch base with @petemill when he's back to see if it needs any additional follow-up.